conanfile.py

```diff
@@ -83,7 +90,14 @@ class wxWidgetsConan(ConanFile):
                "webview": True,
                "xml": True,
                "xrc": True,
-               "cairo": True
+               "cairo": True,
+               "help": True,
+               "html_help": True,
+               "url": True,
+               "protocol": True,
+               "fs_inet": False,
```

Comment:
If I saw that correctly, then upstream has this option enabled by default, right?

Reply:
you mean `fs_inet`?

Reply:
Yes, all options added via `wx_option` seem to have a default value of `True`

https://github.com/wxWidgets/wxWidgets/blob/master/build/cmake/options.cmake#L150
https://github.com/wxWidgets/wxWidgets/blob/master/build/cmake/functions.cmake#L733

Reply:
If possible we choose the default values as they are upstream. Would it be okay to change `fs_inet` per default to True?

Reply:
I think it's OK